.circleci/config.yml

```diff
@@ -75,15 +75,19 @@ jobs:
   test-3.8-signac-1.0:
     <<: *test-template
     environment:
-      SIGNAC_VERSION: "~=1.0.0"
+      SIGNAC_VERSION: "signac~=1.0.0"
   test-3.8-signac-1.3:
     <<: *test-template
     environment:
-      SIGNAC_VERSION: "~=1.3.0"
+      SIGNAC_VERSION: "signac~=1.3.0"
   test-3.8-signac-1.4:
     <<: *test-template
     environment:
-      SIGNAC_VERSION: "~=1.4.0"
+      SIGNAC_VERSION: "signac~=1.4.0"
+  test-3.8-signac-latest:
+    <<: *test-template
+    environment:
+      SIGNAC_VERSION: "git+ssh://git@github.com/glotzerlab/signac.git"
```

Comment:
@bdice do we have the appropriate SSH keys set up for this to work (since you're cloning over SSH, not HTTP)?

Reply:
This was what we used before, so I know it works as-is. I don't believe any special configuration or keys are required because this is a public repository.

Reply:
OK assuming it's working then this looks fine to me.